sql/core/src/test/resources/sql-tests/analyzer-results/hll.sql.out

```diff
@@ -0,0 +1,182 @@
+-- Automatically generated by SQLQueryTestSuite
+-- !query
+CREATE TABLE t1 USING JSON AS VALUES (0), (1), (2), (2), (2), (3), (4) as tab(col)
+-- !query analysis
+org.apache.spark.sql.catalyst.analysis.TableAlreadyExistsException
```

Comment:
I guess this is not an expected exception and error class apparently.

Reply:
Looks like this needs to be fixed :)

Reply:
You're both right, it looks like this is an independent issue. I filed a separate https://issues.apache.org/jira/browse/SPARK-44123 for myself to fix it.

Reply:
It should be really easy to fix here. I believe you can do it in two different ways:
1. Add a `DROP TABLE IF EXISTS t1` all before this one.
2. Replace the `CREATE TABLE` command with `CREATE OR REPLACE TABLE t1...`.